src/app/app.server.ts

```diff
@@ -0,0 +1,12 @@
+import express, {Express} from 'express';
+import { router as cakeRouter } from './cake/cake.router';
+
+const app: Express = express();
+app.use('/cakes', cakeRouter);
+
+export const initServer = (port: number) => (
+	app.listen(port, () => {
+		console.log('Server listen on port', port);
+	})
+);
+
```

Comment:
Use the same tabulation format for all the files 😊  some of the files are with 2 spaces and others with 4 spaces.